docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -185,6 +202,23 @@ type ExpoAssetHeaderDictionary = {
 
 - `assetRequestHeaders`: MAY contain a dictionary of header (key, value) pairs to include with asset requests. Key and value MUST both be strings.
 
+### Directive Body
+
+Defined as JSON conforming to both the following `Directive` definition expressed in [TypeScript](https://www.typescriptlang.org/) and the detailed descriptions for each field:
+
+```typescript
+type Directive = {
+  directiveType: string;
+  directivePayload?: { [key: string]: any };
```

Comment:
Usually I like context-insensitive names in codebases but in a protocol the shorter names fit better IMO, what do you think?
```suggestion
  type: string;
  data?: { [key: string]: any };
```